internal/campaigns/types.go

```diff
@@ -1370,3 +1370,25 @@ type ChangesetSyncData struct {
 func unixMilliToTime(ms int64) time.Time {
 	return time.Unix(0, ms*int64(time.Millisecond))
 }
+
+// FindMergeCommit will return the merge commit from the given set of events, stopping
+// on the first it finds.
+// It returns an empty string if none are found.
+func FindMergeCommit(events []*ChangesetEvent) string {
```

Comment:
I think it's clearer to add either `ID` or `OID`, since we don't return the full commit object:

```suggestion
func FindMergeCommitID(events []*ChangesetEvent) string {
```